src/icons/icon-base/_IconBase.tsx

```diff
@@ -6,13 +6,14 @@ import { type IconProps } from 'src/types/IconProps';
 type Props = { children: ReactNode; viewBox?: string } & IconProps;
 
 export const IconBase = forwardRef<SVGSVGElement, Props>(
-  ({ children, className, color, size = 24, viewBox }, ref) => (
+  ({ children, className, color, inline, size = 24, viewBox }, ref) => (
     <svg
       ref={ref}
       className={className}
       color={color && `${theme[color]}`}
       fill="none"
       height={size}
+      style={{ display: inline ? 'inline' : 'block' }}
```

Comment:
I don't know if we want to deal with `display: inline;`. There are so many downsides for using `inline`, I would prefer to use `display: inline-block` instead 🤔

Reply:
@goodpickle @evad1n Y'all approved this previously. What do you think about Thien's comment? Should we switch it to inline-block?